src/AvtaleSide/ArbeidsoppgaverSteg/OppgaveKort/OppgaveKort.tsx

```diff
@@ -24,18 +24,20 @@ class OppgaveKort extends React.Component<Props, State> {
         this.setState({ iEndreModus: modus });
     };
 
-    lagreOppgave = (oppgave: Oppgave) => {
+    lagreOppgave = async (oppgave: Oppgave) => {
+        await this.props.lagreOppgave(oppgave);
         this.settEndreModus(false);
-        this.props.lagreOppgave(oppgave);
     };
 
     endreOppgave = () => {
         this.settEndreModus(true);
     };
 
     slettOppgave = () => {
-        if (window.confirm("Er du sikker på at du vil slette denne oppgaven?")) {
-            this.props.slettOppgave(this.props.oppgave);
+        if (
```

Comment:
window.confirm('Er du sikker på at du vil slette denne oppgaven?') ? this.props.slettOppgave(this.props.oppgave) : null;